qualang_tools/results/README.md

```diff
@@ -6,20 +6,20 @@ The fetching tool has an API to easily fetch data from the stream processing.
 
 First the results handle needs to be initiated by specifying the QM job and the list of data to be fetched. 
 These values must correspond to results saved in the stream processing. A flag is also avalable with two options:
-- `flag="wait_for_all"` will wait until all values were processed for all named results before fetching.
-- `flag="live"` will fetch data one by one for all named results for live plotting purposes.
+- `mode="wait_for_all"` will wait until all values were processed for all named results before fetching.
+- `mode="live"` will fetch data one by one for all named results for live plotting purposes.
 
 ### Usage example
 
  
 ```python
-from qualang_tools.results import result_tool
+from qualang_tools.results import fetching_tool
 
 qmm = QuantumMachinesManager()

```

Comment:
do you need to specify the IP in the example?

Reply:
Right I should add it for good measure